Library/Homebrew/test/cli/named_args_spec.rb

```diff
@@ -176,18 +176,18 @@ def setup_unredable_cask(name)
     end
   end
 
-  describe "#to_kegs" do
+  describe "#to_default_kegs" do
     before do
       (HOMEBREW_CELLAR/"foo/1.0").mkpath
       (HOMEBREW_CELLAR/"bar/1.0").mkpath
     end
 
-    it "resolves kegs with #resolve_kegs" do
-      expect(described_class.new("foo", "bar").to_kegs.map(&:name)).to eq ["foo", "bar"]
+    it "resolves kegs with #resolve_default_keg" do
+      expect(described_class.new("foo", "bar").to_default_kegs.map(&:name)).to eq ["foo", "bar"]
     end
 
     it "when there are no matching kegs returns an array of Kegs" do
-      expect(described_class.new.to_kegs).to be_empty
+      expect(described_class.new.to_default_kegs).to be_empty
     end
   end
```

Comment:
Should this be added as an additional`describe` instead? That way we test `to_kegs` and `to_default_kegs`